core/src/main/java/io/varhttp/ControllerExecution.java

```diff
@@ -45,7 +47,16 @@ public void execute(ControllerContext context) {
 			filters.add(filterExecution);
 			Iterator<Object> iterator = filters.iterator();
 			VarServletFilterChain chain = new VarServletFilterChain(context, iterator.next(), iterator);
+			if (context.response().getHeader("Content-Type") == null) {
+				Enumeration<String> acceptHeaders = context.request().getHeaders("Accept");
```

Comment:
If accepts is emtpy should we add *.* to the context

Reply:
we should add * to the context, i think